libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -242,16 +253,31 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
         let _stepId = 0;
         this.steps.forEach((step) => {
             if (step.content) {
-                step.content.tallContent = false;
+                if (!step.isSummary) {
+                    step.content.tallContent = false;
+                }
                 step.content.wizardContentId = _stepId.toString();
             }
             step._stepId = _stepId;
             _stepId++;
             step.finalStep = false;
-            if (step.visited && step.branching && step.status === CURRENT_STEP_STATUS) {
-                step.content.nextStep._getElRef().nativeElement.style.display = 'inherit';
-            } else if (step.completed && this.appendToWizard) {
+            /*
+             If the step is completed and appendToWizard is true, hide the nextStep button, unless it's the last step,
+             or if there's a summary and it is the second to last step
+             */
+            if (
+                step.completed &&
+                this.appendToWizard &&
+                (this.steps.last.isSummary
+                    ? step !== this.steps.toArray()[this.steps.length - 2]
+                    : step !== this.steps.last)
```

Comment:
Coud you create variable from this? ex. `lastNonSummaryStep`, It's quite hard to read this condition